sysutils/AutoRaise/Portfile

```
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sbmpost AutoRaise 1.5 v
+
+categories          sysutils
+platforms           macosx
+
+license             {GPL-2 GPL-3}
```

The license header in the source code file says:
```
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
```
So this should be:
```suggestion
license             GPL-3+
```

---

So if a source allows GPL-2 or higher, at MacPorts we choose GPL-3 by default? Or should that rather be "GPL-2+"?

---

My mistake; I meant `GPL-2+`.